sample/NotificationHub-CSharp/function.json

```diff
@@ -0,0 +1,14 @@
+﻿{
+  "bindings": [
+    {
+      "type": "queueTrigger",
+      "direction": "in",
+      "queueName": "samples-notificationhub-csharp"
+    },
+    {
+      "type": "notificationhub",
+      "direction": "out",
+      "name": "messageProperties"
```

Comment:
messageProperties - is JSON string with key value pairs that will be used to build templateNotification